ledger/acctupdates.go

```diff
@@ -1195,6 +1195,7 @@ func (au *accountUpdates) lookupResource(rnd basics.Round, addr basics.Address,
 	for {
 		currentDbRound := au.cachedDBRound
 		currentDeltaLen := len(au.deltas)
+		rnd := currentDbRound + basics.Round(currentDeltaLen)
```

Comment:
could we keep rnd as an argument to this low-level method, but have the callers that wrap it up for public consumption pass in latest() instead of hard-coding it here?